docs/source/guides/installation_guide.md

```diff
@@ -103,6 +104,21 @@ Here is the recommended procedure for setting up `nucypher` in this fashion:
     import nucypher
     ```
 
+### Standard Docker Install
+1. Install [Docker](https://docs.docker.com/install/)
+
+2. (Optional) Follow these post install instructions: https://docs.docker.com/install/linux/linux-postinstall/
+
+3. Get the latest nucypher image:
+
+  `(maybe sudo) docker pull nucypher/nucypher:latest`
```

Comment:
Eh, I think you probably do it wrong if you need sudo?

Reply:
✅